devel/ansible/roles/dev/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Install dev packages
+  dnf:
+      name: "{{ item }}"
+      state: present
+  with_items:
+      - python
+      - python-devel
+      - python-webhelpers
+      - fedmsg
+      - python-kitchen
+      - python-fedora
+      - python-bugzilla
+      - koji
+      - xapian-bindings-python
+      - diffstat
+      - fedpkg
+      - svn
+      - wget
+      - python-xappy
+      - python-webob
+      - moksha
+      - TurboGears2
+      - python-dogpile-cache
+      - python-dogpile-core
+      - python-memcached
+      - memcached
+      - python-markdown
+
+- name: Install python packages
+  pip:
+      name: "{{ item }}"
+      state: present
+  with_items:
+      - python-appstream
+      - gearbox
+
+- command: "{{ item }}"
+  become: yes
+  become_user: vagrant
+  args:
+    chdir: /vagrant/
+  with_items:
+    - rm -rf /vagrant/xapian/
```

Comment:
I would recommend not storing this file in /vagrant, especially since it gets removed every time. Since vagrant is served over sshfs, it'll perform slowly. You could use /tmp, or if that's too small you could use /home/vagrant/.